main/options.c

```diff
@@ -3438,13 +3273,19 @@ static int accept_only_dot_ctags(const struct dirent* dent)
 	return 0;
 }
 
+static int local_insenstive_alphasort(const struct dirent ** a,
```

Comment:
Do you mean "locale" instead of "local"?

Reply:
Thank you. The function name is wrong. It should be... my_alphasort. I cannot remember why I named the function local_insenstive_alphasort.

I will make a fixup commit.